src/components/pages/Header.js

```diff
@@ -0,0 +1,21 @@
+import './Style.css';
+
+const Header = () => (
+  <header>
+    <nav>
```

Comment:
- [ ] [Optional] `Header.js` suppose to be in `components` folder directly or put it inside `containers` folder, not inside `pages` folder. and It would be better professionally if `pages` and `containers` folders not be child folders for the `components` folder. the file structure suppose to be like that:
```
src
├── components
├── containers
└── pages
```